<launch>
    <env name="emulated_sonar" value="true" />
    <node pkg="ping360_sonar" type="ping360_node" name="ping360_node" output="screen">
        <param name="device" value="/dev/ttyUSB0"/>
        <param name="baudrate" value="115200"/>
        <param name="debug" value="True"/>
        <param name="imgSize" value="500"/>
        <param name="gain" value="0"/>
        <param name="step" value="1"/>
        <param name="numberOfSamples" value="200"/>
        <param name="transmitFrequency" value="740"/>
        <param name="sonarRange" value="1"/>
        <param name="speedOfSound" value="1500"/>
        <param name="queueSize" value="1"/>
        <param name="threshold" value="100"/>
    </node>
</launch>